app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
-  belongs_to :author_id
-  belongs_to :post_id
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  private
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] [OPTIONAL] it would be better if you used `increment` to update the post counter, it's easier and street forward 
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```